app/js/actions/fetchOrder.js

```diff
@@ -0,0 +1,16 @@
+import 'babel-polyfill';
+import axiosInstance from '../config';
+
+import { FETCH_ORDERS } from './actionTypes';
+
+const fetchOrders = (
+  uri,
```

Comment:
The name of this file is not the same as its `export-default`, `fetchOrder` vs `fetchOrders`.